src/main/java/metrics_influxdb/InfluxdbHttp.java

```diff
@@ -101,6 +101,26 @@ public InfluxdbHttp(String host, int port, String path, String database, String
 		);
 	}
 
+	/***
+	 *
+	 * @param version the Influxdb version, currently builds a different url for 0.9 versions, else defaults to the earlier url
+	 * @throws Exception
+	 */
+	public InfluxdbHttp(String host, int port, String database, String username, String password, TimeUnit timePrecision, String version) throws Exception  {
+		if("0.9".equals(version)){
+			this.url = new URL("http", host, port,
+					"/write?u=" + username + "&p=" + password);
+			this.jsonBuilder = new JsonBuilderV9(database, toTimePrecision(timePrecision));
+		} else {
```

Comment:
What exactly is the else case? Does it really cover *every* other version? Also future ones?
I think you should be more specific about the versions that can be handled here and throw an IllegalArgumentException if the version does not match.